Assets/Scripts/Player/Controllers/AbilityController.cs

```diff
@@ -0,0 +1,45 @@
+using Abilities;
+using Extensions;
+using UnityEngine;
+
+namespace Player.Controllers
+{
+    [RequireComponent(typeof(PlayerStats))]
+    public class AbilityController : MonoBehaviour
+    {
+        [Expandable] [SerializeField] public Ability StandardAbility;
```

Comment:
If these are going to remain public then the `[SerializeField]` attribute is not needed.